docs/deployment/flyio.md

```diff
@@ -356,7 +356,11 @@ ALLOWED_HOSTS = os.getenv("DJANGO_ALLOWED_HOSTS", "*").split(",")
 
 CSRF_TRUSTED_ORIGINS = os.getenv("DJANGO_CSRF_TRUSTED_ORIGINS", "").split(",")
 
-EMAIL_BACKEND = "django.core.mail.backends.console.EmailBackend"
+EMAIL_BACKEND = "django.core.mail.backends.smtp.EmailBackend"  # Use this for sending emails via an SMTP service. 
+
+"""
+If you are in a development environment and do not want to send real emails, use `django.core.mail.backends.console.EmailBackend` instead.
+"""
```

Comment:
We don’t want to do this kind of code change here. We want the tutorial to keep working for people who don’t want to set up an email backend.

Reply:
@thibaudcolas This code snippet is the file `mysite/settings/production.py`. People who don’t want to set up an email backend are unable to send email on their production site – they won’t see them anyway. I’d say it’s necessary to configure email if the demo site contains a contact form...

Reply:
@thibaudcolas Thank you for your feedback!

I understand that keeping the tutorial simple and functional without requiring email setup is important, so I’ll revert the code snippet to avoid enforcing SMTP configuration.

However, as @garraflavatra mentioned regarding the need for email setup on demo sites with contact forms, I’m considering a documentation update to clarify that ConsoleBackend is suitable for local testing and SMTPBackend should be used in production if email functionality is needed.

Please let me know your thoughts. Apologies for the delay—I’ve been a bit occupied with academics.